content/en/docs/components/model-registry/overview.md

```diff
@@ -0,0 +1,99 @@
++++
+title = "Overview"
+description = "An overview for Kubeflow Model Registry"
+weight = 10
+                    
++++
+
+{{% alpha-status
+  feedbacklink="https://github.com/kubeflow/model-registry" %}}
+
+## What is Model Registry?
+
+A model registry is an important component in the life cycle of AI/ML models, an integral component for any MLOps platform and for ML workflows.
```

Comment:
Do you want to re-use the ML lifecycle diagram that we are trying to add here: https://github.com/kubeflow/website/pull/3728 ?
E.g. we already re-used it in the Training Operator and Katib: https://www.kubeflow.org/docs/components/training/overview/#why-training-operator-

Reply:
To me, it sounds like this page can eventually accomodate more than 1 diagram... as can be seen in this PR thread, there are several concepts which might need expression in visual form:
- Model Registry being a central and pivotal component in the MLOps lifecycle, showcasing both inner and outer loop for MLOps
- Model Registry as "single pane of glass"
- Model Registry as involved in each task/phase of the MLOps workflow

So overall I believe we should eventually accomodate more than a single diagram, as in a single image sounds to me a bit difficult to be exactly like "everybody" want to see :) wdyt?
Should we progress with the agreement we can expand with more diagram as a next iteration?

Reply:
I agree with you @tarilabs, I also want to modify this diagram to show that model registry plays role in Model Development, Model Tuning, Model Training, Model Serving stages.

Reply:
tracked in: https://github.com/kubeflow/model-registry/issues/109

marking as resolved this thread accordingly.